LEGO1/viewroi.h

```diff
@@ -0,0 +1,47 @@
+#ifndef VIEWROI_H
+#define VIEWROI_H
+
+#include "orientableroi.h"
+#include "tgl.h"
+#include "viewlodlist.h"
+
+/*
+	ViewROI objects represent view objects, collections of view objects,
+	etc. Basically, anything which can be placed in a scene and manipilated
+	by the view manager is a ViewROI.
+*/
+class ViewROI : public OrientableROI {
```

Comment:
In the 1996 source the base class is `RelativeROI` - has this been changed on purpose?

Reply:
Somewhere along the line one of OrientableROI or RelativeROI was removed, or they got merged.  What still exists more closely matches OrientableROI, so I went with that name.